### PR TITLE
Raise exception in pathfixer if path is not ascii

### PR DESCRIPTION
**Pull request by wesleybl**
When migrating from other CMSs to Plone, paths may have non-ascii characters. I think the most correct thing to do is to raise an exception, so that the person who is migrating does the analysis and decides what to do. We cannot simply remove non-ascii characters. Plone itself doesn't do this with ids. The title `testç` becomes `testc` and not `test`. Rich text fields can have links pointing to the path with non-ascii characters. These links can break. A broader analysis of removing non-ascii characters should be done.

Also don't encode path in Python 3.
This that did the path to become a byte. This caused problem in other pipelines.

### CODE REVIEW

File: src/plone/app/transmogrifier/utils.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""Functions utils."""
+from Products.CMFPlone.utils import safe_unicode
```
Comment (jensens) [inline src/plone/app/transmogrifier/utils.py:3]: `safe_unicode` is deprecated - use `safe_text`.

Comment (wesleybl) [reply src/plone/app/transmogrifier/utils.py]: @jensens I changed to safe_text.

Comment (wesleybl) [reply src/plone/app/transmogrifier/utils.py:3]: @jensens `Plone < 5.2` doesn't have `safe_text`. See:

https://github.com/collective/plone.app.transmogrifier/runs/3196672035?check_suite_focus=true#step:9:819

 I went back to `safe_unicode`

Comment (jensens) [reply src/plone/app/transmogrifier/utils.py:3]: Indeed, please conditional import it.

Comment (wesleybl) [reply src/plone/app/transmogrifier/utils.py]: Done.

**Comment by wesleybl** [comment]
@jensens  @mauritsvanrees any impediments to merge this PR? If afraid to raise exception, I can do another PR with just the Python 3 encode part:

> Also don't encode path in Python 3.
This that did the path to become a byte. This caused problem in other pipelines.

**Pull request merged**